### PR TITLE
rpc: getrpcinfo docs

### LINKED ISSUE #15731: Add getrpcinfo doc
https://github.com/bitcoin/bitcoin/issues/15555#issuecomment-479346264

### PR DESCRIPTION
**Pull request by benthecarman**
Fixes #15731

### CODE REVIEW

File: src/rpc/server.cpp [hunk 1]
```diff
@@ -297,8 +297,17 @@ static UniValue getrpcinfo(const JSONRPCRequest& request)
             RPCHelpMan{"getrpcinfo",
                 "\nReturns details of the RPC server.\n",
                 {},
-                RPCResults{},
-                RPCExamples{""},
+                RPCResult{
+            "{ (json object with active commands)\n"
+            "  { (json object with information about an active command)\n"
```
Comment (promag) [inline src/rpc/server.cpp]: Should have the key `"active_commands":` saying it's a json array.

Comment (benthecarman) [reply src/rpc/server.cpp]: fixed

**Review by promag — COMMENTED** [review]
The output should be something like
```
Result:
{
 "active_commands" (array) All active commands
   [                 
     {             (object) Information about an active command
       "method"    (string) The name of the RPC command
       "duration"  (numeric) The running time in microseconds
     },...
   ]
}
```

**Comment by promag** [comment]
LGTM.

File: src/rpc/server.cpp [hunk 2]
```diff
@@ -297,8 +297,20 @@ static UniValue getrpcinfo(const JSONRPCRequest& request)
             RPCHelpMan{"getrpcinfo",
                 "\nReturns details of the RPC server.\n",
                 {},
-                RPCResults{},
-                RPCExamples{""},
+                RPCResult{
+            "{\n"
```
Comment (laanwj) [inline src/rpc/server.cpp]: Maybe indent this line and the strings below it four to the right?

**Comment by laanwj** [comment]
utACK f4b7a2f2055fa52d5e02c37bed7b46599b960e2e

**Comment by fanquake** [comment]
Bckported in #15790.

**Pull request merged**